README.adoc

```diff
@@ -52,6 +52,35 @@ and configure Maven to run it after building the application.
 [role='command']
 include::{common-includes}/gitclone.adoc[]
 
+=== Try what you'll build
+To try out the application, first navigate to the `finish` directory and then run the following Maven goals to build the application and run it inside Open Liberty:
```

Comment:
Take a look at the common "try what you'll build" adoc for the text that are used: https://github.com/OpenLiberty/guides-common/blob/master/trywhatyoubuild-intro.adoc. See if you can just use the common adoc. Even if not, try to use the similar text. There's also the "end" adoc too.